client/meteor-analytics.js

```diff
@@ -150,3 +150,10 @@ Meteor.startup(function () {
     Tracker.autorun(trackLogins);
   }
 });
+
+
+analytics = this.analytics;
+try {
```

Comment:
You just made the world a slightly better place :)
On 22 May 2016 12:13, "Mickael Lecoq" <notifications@github.com> wrote:

> In client/meteor-analytics.js
> <https://github.com/okgrow/analytics/pull/107#discussion_r64146753>:
>
> > @@ -150,3 +150,10 @@ Meteor.startup(function () {
> >      Tracker.autorun(trackLogins);
> >    }
> >  });
> > +
> > +
> > +analytics = this.analytics;
> > +try {
>
> done ;)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/okgrow/analytics/pull/107/files/7ced6f3af869c675b0a1351d093bf2c978a17a53#r64146753>
>